pyaedt/modeler/Primitives3D.py

```diff
@@ -786,3 +790,316 @@ def get_3d_component_object_list(self, componentname):
+        self.multiparts.append(person1)
+        return person1
+
+    @aedt_exception_handler
+    def add_vehicle(self, actor_folder, speed=0, global_offset=[0,0,0], yaw=0, pitch=0,roll=0):
+        """Add a Moving Vehicle Multipart from 3D Components. It requires a json file in the folder containing vehicle infos.
+
+         .. code-block:: json
+
+            {
+                "name": "vehicle3",
+                "version": 1,
+                "type":"mustang",
+                "class":"vehicle",
+                "xlim":["-1.94","2.8"],
+                "ylim":["-.91",".91"],
+                "parts": {
+                    "wheels_front": {
+                        "comp_name": "wheels_front.a3dcomp",
+                        "rotation_cs":["1.8970271810532" ,"0" ,"0.34809664860487"],
+                        "tire_radius":"0.349",
+                        "rotation_axis":"Y"
+                        },
+                    "wheels_rear": {
+                        "comp_name": "wheels_rear.a3dcomp",
+                        "rotation_cs":["-0.82228746728897" ,"0","0.34809664860487"],
+                        "tire_radius":"0.349",
+                        "rotation_axis":"Y"
+                        },
+                    "body": {
+                        "comp_name": "body.a3dcomp",
+                        "rotation_cs":null,
+                        "tire_radius":null,
+                        "rotation_axis":null
+                        }
+                }
+            }
+
+        Parameters
+        ----------
+        actor_folder: str
+            Path to the actor folder. It must contain a json settings file and a 3dcomponent (.a3dcomp)
+        speed:  float, Optional
+            Object movement speed with time
+        global_offset: list, Optional
+            Offset from Global Coordinate System.
+        yaw: float, Optional
+            Yaw Rotation from Global Coordinate System.
+        pitch: float, Optional
+            Pitch Rotation from Global Coordinate System.
+        roll: float, Optional
+            Roll Rotation from Global Coordinate System.
+
+        Returns
+        -------
+        :class: `pyaedt.modeler.MultiPartComponent.Vehicle`
+
+        """
+        self._initialize_multipart()
+
+        if not self._check_actor_folder(actor_folder):
+            return False
+        vehicle = Vehicle(actor_folder, speed=speed)
+        vehicle.offset = global_offset
+        vehicle.yaw = self._arg_with_dim(yaw, "deg")
+        vehicle.pitch = self._arg_with_dim(pitch, "deg")
+        vehicle.roll = self._arg_with_dim(roll, "deg")
+        vehicle.insert(self._parent)
+        self.multiparts.append(vehicle)
+        return vehicle
+
+    @aedt_exception_handler
+    def add_bird(self, actor_folder, speed=0, global_offset=[0, 0, 0], yaw=0, pitch=0,
+                 roll=0, flapping_rate=50):
+        """Add a Bird Multipart from 3D Components. It requires a json file in the folder containing bird infos.
```

Comment:
Short description should always contain one sentence.  Add additional information to the extended description in the next "section.

```suggestion
        """Add a Bird Multipart from 3D Components.

        It requires a json file in the directory containing bird information.
```